browser-runtime/src/encode-decode.ts

```diff
@@ -138,11 +146,11 @@ export function encode(typeTable: DeepReadonly<TypeTable>, path: string, type: D
   } else if (simpleTypes.indexOf(type) >= 0) {
     return simpleEncodeDecode(path, type, value);
   } else if (type === "bytes") {
-    if (!(value instanceof Buffer)) {
+    if (!(value instanceof ArrayBuffer)) {
       throw new ParseError(path, type, value);
     }
 
-    return value.toString("base64");
+    return btoa(String.fromCharCode.apply(null, (new Uint8Array(value) as unknown) as number[]));
```

Comment:
Nitpick: o `.apply` foi pra evitar usar spread?
```suggestion
    return btoa(String.fromCharCode(...(new Uint8Array(value) as unknown) as number[]));
```